packages/mask/src/plugins/Tips/components/TipButton.tsx

```diff
@@ -94,6 +100,27 @@ export const TipButton: FC<Props> = ({
         return wallets
     }, [receiver])
 
+    const walletsFromCloud = useMemo(() => {
+        if (kv?.ok) {
+            if (!kv.val.proofs.length) return null
```

Comment:
remove this line?

Reply:
If a user haven't done any setting, it would be empty, then we set it `null`